alot/defaults/alot.rc.spec

```diff
@@ -17,6 +17,10 @@ input_timeout = float(default=1.0)
 # .. note:: this config setting is equivalent to, but independent of, the 'search.exclude_tags' in the notmuch config.
 exclude_tags = force_list(default=list())
 
+# Show quotes be parsed
+parse_quotes = boolean(default=True)
+quote_symbol = string(default='>')
```

Comment:
why not use the existing `quote_prefix`?

Reply:
I didn't see it before!  

However, I notice that it has a space added to it. That would mean that a line like:

```
>> Level 2 quote level
```

wouldn't get interpreted correctly, because the regex `r'^ *({} *){{{}}}'.format(symbol, quote_level)` wouldn't match. It would still be possible to change it but the default wouldn't work really well. 

I'm not sure in what context  `quote_prefix` is used. A first solution would be to remove the space and accommodate for this wherever it is used. A second could be to rename my `quote_symbol` to something more explicit, like `quotation_regex_detection`.

Reply:
I don't think this new option is necessary. The two most commonly used characters for quoting text are `>` and `|`, and I've only ever seen the bracket being used ([wikipedia reference](https://en.wikipedia.org/wiki/Posting_style#Quoted_line_prefix)).

It would make the PR leaner to assume only those two characters highlight quotes in messages, plus it would avoid some weird unit tests verifying that the code behaves as expected when using `[Aa]` for `quote_symbol`.